apis/openapi/slack.com/main/1.7.0/openapi.json

```diff
@@ -23225,27 +23304,30 @@
       }
     },
     "/views.open": {
-      "get": {
-        "parameters": [
-          {
-            "description": "Exchange a trigger to post to the user.",
-            "in": "query",
-            "name": "trigger_id",
-            "required": true,
-            "schema": {
-              "type": "string"
-            }
-          },
-          {
-            "description": "A [view payload](/reference/surfaces/views). This must be a JSON-encoded string.",
-            "in": "query",
-            "name": "view",
-            "required": true,
-            "schema": {
-              "type": "string"
+      "post": {
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "trigger_id": {
+                    "type": "string",
+                    "description": "Exchange a trigger to post to the user."
+                  },
+                  "view": {
+                    "type": "object",
+                    "description": "A view payload."
+                  }
+                },
+                "required": [
+                  "trigger_id",
```

Comment:
`trigger_id` doesn't seem to be required according to https://docs.slack.dev/reference/methods/views.open/